config/environments/test.rb

```diff
@@ -49,7 +49,7 @@
   # Raise exceptions for disallowed deprecations.
   config.active_support.disallowed_deprecation = :raise
 
-  # Tell Active Support which deprecation messages to disallow.
+  # Tell Active Support which deprecation notes to disallow.
```

Comment:
revert this

Reply:
basically changed the table name from message to notes. so that these changes occurs.